crates/oxc_ast/src/ast/comment.rs

```diff
@@ -1,5 +1,6 @@
 #![warn(missing_docs)]
-use oxc_allocator::CloneIn;
+use bitflags::bitflags;
+use oxc_allocator::{Allocator, CloneIn};
```

Comment:
This is a total nit, but would you mind putting the `use bitflags::bitflags;` in it's own block above the `oxc_*` dependencies? See #7883 for the rationale.

```suggestion

use bitflags::bitflags;

use oxc_allocator::{Allocator, CloneIn};
```